src/main/java/org/mqttbee/api/mqtt/MqttWebsocketConfig.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2018 The MQTT Bee project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.mqttbee.api.mqtt;
+
+import org.mqttbee.annotations.NotNull;
+
+public class MqttWebsocketConfig {
```

Comment:
Mixture of builder and config entity.
Split in MqttWebsocketConfig and MqttWebsocketConfigBuilder

Reply:
Seems like overkill for setting one string (serverPath)

Reply:
As this is part of the API it should be consistent and allow us to add future config options for websockets without unnecessarily breaking "backwards compatibility" (we can just add a new method here)
In the meantime we could offer a convenience method MqttWebsocketConfig.create(String serverPath)

Reply:
ok, but also for later